classes/04class/exercises/c04-iac01/gpavelar/ec2.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_instance" "test-ec2-instance" {
+  ami                         = "ami-0a58e22c727337c51"
```

Comment:
tip: it can be a variable passed in or you can use `data sources` to dynamically fetch the latest AMI